### PR TITLE
Fixing sample size error for Attribute charts 

### PR DESCRIPTION
**Pull request by tomishay**

### CODE REVIEW

**Comment by vandenman** [comment]
Once this is ready for review, please add me as a reviewer

**Comment by tomishay** [comment]
Will do!

File: R/AttributeCharts.R [hunk 1]
```diff
@@ -120,6 +120,16 @@ if(options$Attributes == "Defectives"){
   if (!ready)
     return()
 
+  .hasErrors(
```
Comment (vandenman) [inline R/AttributeCharts.R:123]: is it necessary to make 2 identical calls to the function `.hasErrors`? Why no create a function, say `.attributeChartsErrorCheck`, and then call that function once before you call these two functions?

Comment (tomishay) [reply R/AttributeCharts.R:123]: I can make a function and then call it twice, but this check is only necessary for these two specific charts, not to the entire analysis.

**Comment by vandenman** [comment]
So the r-chart changed quite a bit. Can you verify that the changes are intended? If the changes are fine, please update the test figure.

**Review by vandenman — CHANGES_REQUESTED** [review]
Changes are ok, but test needs to be fixed

**Pull request closed without merging**